onadata/libs/tests/utils/test_export_builder.py

```diff
@@ -52,6 +51,8 @@ class TestExportBuilder(TestBase):
             'name': 'Abe',
             'age': 35,
             'tel/telLg==office': '020123456',
+            '_review_status': 'Active',
```

Comment:
I know it is a test, but we dont have `Active` as a possible review status.  Should we avoid any possibly misleading even in tests by using the actual expected value(s)?